views/default/anypage/content.php

```diff
@@ -13,4 +13,6 @@
 	return;
 }
 
-echo $page->description;
+echo elgg_view('output/longtext', array(
```

Comment:
Maybe add a `$vars['raw']` option to just output the description property? I can update my plugin as needed.